pkg/controller/label_event_handler_test.go

```diff
@@ -0,0 +1,62 @@
+package controller
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func TestLabelMapper(t *testing.T) {
+
+	t.Run("resource with expected label", func(t *testing.T) {
+		// given
+		objMeta := metav1.ObjectMeta{
+			Name: "bar",
+			Labels: map[string]string{
+				"owner": "foo",
```

Comment:
could you please add more labels to verify that the mapper is able to find it there (I know that it does - just to be sure :-)) 
```suggestion
				"type": "che",
				"owner": "foo",
				"revision": "123",
```